src/source_map.js

```diff
@@ -29,11 +29,11 @@ export class SourceMap {
   }
 
   getCombinedSource() {
-    return Object.values(this.sourceFiles).join('\n');
```

Comment:
Since we're already using `babel`, is there no way for `babel` to include a `Object.values` polyfill? I remember seeing this somewhere. 🤔

Reply:
There might be a Babel plugin, but this is simple enough.  Lemme know if you want me to dig deeper.

Reply:
I was mostly worried about us needing `Object.values` elsewhere at some point.

I suppose we could solve that if/when we reach that point.